source/user-manual/reference/ossec-conf/syscheck.rst

```diff
@@ -739,25 +743,45 @@ this parameter is ignored.
 
 **interval**
 
-Specifies the elapsed time between every inventory synchronization.
+Specifies the initial number of seconds between every inventory synchronization. If synchronization fails the value will be duplicated until it reaches the value of ``max_interval``.
```

Comment:
Don't say "the initial number of seconds" because it is allowed to set a suffix with the desired scale (seconds, minutes, hours, etc.)